docs/static/config-details.asciidoc

```diff
@@ -1,20 +1,25 @@
+[[jvm-settings]]
+=== JVM settings
+
+Configure the jvm settings in the `jvm.options` <<settings-files,settings file>>.
+
 [[heap-size]]
-=== Setting the heap size
+==== Setting the JVM heap size
 
-Set the jvm heap size in the `jvm.options` <<config-setting-files,configuration
-file>>.
+Here are some tips for adjusting the JVM heap size:
 
-NOTE: The recommended heap size for typical ingestion scenarios should be no
+// tag::heap-size-tips[]
+* The recommended heap size for typical ingestion scenarios should be no
 less than 4GB and no more than 8GB.
```

Comment:
this is somewhat out of context for this PR but I am not sure about these 4/8 numbers. For me "typical ingestion scenarios" seems too vague and 4/8 seems somewhat arbitrary...

Reply:
@colinsurprenant  Thanks for the review and the comments.  I will open a new issue and tag you on it.